engine/apps/alerts/grafana_alerting_sync_manager/grafana_alerting_sync.py

```diff
@@ -357,7 +356,8 @@ def _create_contact_point_from_payload(self, payload, datasource) -> "GrafanaAle
 
         receiver_config = self._get_receiver_config(is_grafana_datasource, payload)
 
-        GrafanaAlertingContactPoint = apps.get_model("alerts", "GrafanaAlertingContactPoint")
+        from apps.alerts.models import GrafanaAlertingContactPoint
```

Comment:
should we move the imports to the top?

Reply:
Yes, I think imports should be at the top, I'll try to come up with an additional PR for this (+adding a rule to enforce that in CI)